src/main/java/mate/academy/Main.java

```diff
@@ -2,14 +2,13 @@
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Main {
     public static void main(String[] args) {
-        // Feel free to play with AsyncRequestProcessor in this main method if you want
-        ExecutorService executor = null; // Provide implementation that fits your needs
+        ExecutorService executor = Executors.newFixedThreadPool(11);
```

Comment:
I'd suggest using a common thread pool instead of a fixed number, to ensure that this code will be run using reasonable resources in accordance with the hardware where this will be run at